### ISSUE TITLE
Cost-analyser init logs excessively when it can't reach Prometheus

### ISSUE THREAD

**Issue by leosunmo** [kubecost/cost-model#175]
When bringing up Kubecost-cost-analyzer the cost-analyzer-init container logs once every 10 ms that it can't reach Prometheus.

```
kubecost-cost-analyzer-59866c78-blv9c cost-analyzer-init I1008 21:41:25.351443       1 main.go:160] Failed to use prometheus at http://kubecost-prometheus-server.ops with service  in namespace  in zone  : Get http://kubecost-prometheus-server.ops/api/v1/status/config: dial tcp: lookup kubecost-prometheus-server.ops on 100.64.0.10:53: no such host```

**Comment by dwbrown2** [comment]
Thanks for the report @leosunmo! Sounds excessive to me =P 

We'll get this addressed.

**Comment by leosunmo** [comment]
Thank you!

**Comment by dwbrown2** [comment]
@leosunmo this should be resolved in v1.45 (released last night)! Let me know if you have any further issues. 

What else can we get you?! We're always excited for new feature requests

**Comment by leosunmo** [comment]
Deploying 1.45.2 now.
